pdm_build.py

```diff
@@ -0,0 +1,35 @@
+import shutil
+import subprocess
+import sysconfig
+
+
+def pdm_build_hook_enabled(context):
+    # Disable hooks for sdist builds
+    return context.target != "sdist"
+
+
+def pdm_build_initialize(context):
+    context.ensure_build_dir()
+    # Inject compiled binary into scripts/, so it will be picked up to install
+    target = context.root / "scripts" / "bids-validator-deno"
+
+    deno = shutil.which("deno")
+    if deno is None:
+        raise OSError("Deno is not installed or not in PATH")
+
+    subprocess.run(
+        [
+            deno,
+            "compile",
+            "-ERNW",
+            "--allow-run",
+            "-o",
+            str(target),
+            "src/bids-validator.ts",
+        ],
+        check=True,
+    )
+
+    # Add the current platform tag so the wheel is specific to the OS/architecture
+    platform_tag = sysconfig.get_platform().replace("-", "_").replace(".", "_")
+    context.config_settings["--plat-name"] = platform_tag
```

Comment:
@candleindark Updating the tag not to be `py3-none-any` is enough for cibuildwheel to handle this. There are other cases where you can have `py3-none-<platform>`, if you have compiled extensions that don't use the CPython API.

Reply:
this would definitely be cleaner,  but did you try to upload/install those wheels across OSes? It seems that even test.pypi still has our old ones from yesterday? (I think) https://test.pypi.org/project/bids-validator-deno/#files

Reply:
1) The wheels are installed and tested by cibuildwheel. Happy to revisit if the wheels don't install. I tried to build with the oldest images that have a new enough glibc to run deno, so they should be broadly compatible.
1) I don't have access to upload to test.pypi.org. Happy to test if you give me access.